### ISSUE TITLE
Update screens to kivy 2.2 and multiple display size functionality

### ISSUE THREAD

**Issue by ISS-Mimic** [ISS-Mimic/Mimic#384]
- [x] CDH_Screen.kv
- [x] CT_Camera_Screen.kv
- [x] CT_SASA_Screen.kv
- [x] CT_SGANT_Screen.kv
- [x] CT_Screen.kv
- [x] CT_UHF_Screen.kv
- [x] CalibrateScreen.kv
- [x] Crew_Screen.kv
- [x] ECLSS_Screen.kv
- [x] EPS_Screen.kv
- [x] EVA_Main_Screen.kv
- [x] EVA_Pictures.kv
- [x] EVA_RS_Screen.kv
- [x] EVA_US_Screen.kv
- [x] FakeOrbitScreen.kv
- [x] GNC_Screen.kv
- [x] ISS_Screen.kv
- [x] LED_Screen.kv
- [x] MSS_MT_Screen.kv
- [x] MainScreen.kv
- [x] ManualControlScreen.kv
- [x] MimicScreen.kv
- [x] Orbit_Data.kv
- [x] Orbit_Pass.kv
- [x] Orbit_Screen.kv
- [x] RS_Dock_Screen.kv
- [x] RS_Screen.kv
- [x] Robo_Screen.kv
- [x] SPDM1_Screen.kv
- [x] SSRMS_Screen.kv
- [x] Science_Screen.kv
- [x] Settings_Screen.kv
- [x] TCS_Screen.kv
- [x] USOS_Screen.kv
- [x] VV_Screen.kv

**Comment by ISS-Mimic** [comment]
EVA screen needs text adjusting

**Comment by GodspeedDiscvry** [comment]
Checking out on 7" and I have the following:

- giant text on Simulated screen for "OFT2 Loop" and "Disco!".  The disco one should be olkay - but will be a button - and actually OFT2 Loop will probably be a button too.
- orbit data the upper right text is now smaller and does not line up with identifiers
- orbit still does not work (no data displayed)
- US EVA we already discussed some issues here - but the crewlock status in upper right is not showing code that is in GUI instead of EVA screen code (and this was not actually on your list above)
- SGANT is doing something similar in that our transparent png is including a black box that wipes out other stuff (like the back button) when the transparent png rotates
---
Screens not done at the time if this comment:  Russian Docking, Robo, MT/MBS, SSRMS, SPDM, VV - will check those once checked off.  I did however include the C&T subs on my check even though not annotated above.
